src/Abp/Runtime/Validation/Interception/MethodInvocationValidator.cs

```diff
@@ -138,6 +138,11 @@ protected virtual void ThrowValidationError()
         /// <param name="parameterValue">Value to validate</param>
         protected virtual void ValidateMethodParameter(ParameterInfo parameterInfo, object parameterValue)
         {
+            if (_configuration.IgnoredTypes.Any(t => t.IsInstanceOfType(parameterValue)))
```

Comment:
I moved this block here because I think that it should skip validation early before checking for null value.